packages/mui-material-next/package.json

```diff
@@ -58,8 +58,8 @@
   },
   "dependencies": {
     "@babel/runtime": "^7.15.4",
-    "@mui/core": "5.0.0-alpha.48",
-    "@mui/system": "^5.0.1",
+    "@mui/core": "5.0.0-alpha.49",
+    "@mui/system": "^5.0.2",
```

Comment:
Did you manually add the caret or did `yarn release:version` leave it in?

Reply:
I manually added the caret. It should have the caret right (this is the part I'm not sure)?

Reply:
Yep, all good. I don't know why the caret is removed so I was hoping you figured it out somehow.

Reply:
I see, that's still a myth 😂 . I hope I'll figure it out soon.